### PR TITLE
Update a bunch of Github Actions

### PR DESCRIPTION
**Pull request by JBassett**
## Summary

We are using several deprecated Github Actions, this should resolve those issues.

## Screenshots

N/A

## Link to pull request in Documentation repository

N/A

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -112,6 +112,6 @@ jobs:
         fi
     
     - name: Archive Debug Build
-      uses: kittaakos/upload-artifact-as-is@v0
+      uses: actions/upload-artifact@v3
```
Comment (jpelgrom) [inline .github/workflows/pr.yml:115]: The PR flow now includes one file - artifact, 82 MB - instead of all APKs

Comment (jpelgrom) [reply .github/workflows/pr.yml:115]: It's less convenient, but not a major problem. Just wanted to point it out :)

Comment (JBassett) [reply .github/workflows/pr.yml:115]: That action was archived and throwing warnings so figured it was safer to use a more default one.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, too bad there is no action to expand the artifact 😛

**Pull request merged**